modules/installation-creating-image-restricted.adoc

```diff
@@ -25,13 +36,19 @@ the {product-title} version that you install. Use the image versions that match
 your {product-title} version if they are available.
 ====
 
+ifdef::osp[]
 . Download the *{op-system-first} - OpenStack Image (QCOW)*.
+endif::osp[]
+ifdef::vsphere[]
+. Download the *{op-system-first} - vSphere*.
```

Comment:
Should this say, "Download the **Red Hat Enterprise Linux CoreOS (RHCOS) - vSphere** _image_." ?